test/extensions/quic_listeners/quiche/platform/quic_platform_test.cc

```diff
@@ -397,6 +397,8 @@ TEST_F(QuicPlatformTest, QuicheCheck) {
   EXPECT_DEATH({ QUICHE_CHECK_LT(1 + 1, 2); }, "CHECK failed: 1 \\+ 1 \\(=2\\) < 2 \\(=2\\)");
   EXPECT_DEBUG_DEATH({ QUICHE_DCHECK_NE(1 + 1, 2); },
                      "CHECK failed: 1 \\+ 1 \\(=2\\) != 2 \\(=2\\)");
+  EXPECT_DEBUG_DEATH({ QUICHE_DCHECK_NE(nullptr, nullptr); },
+                     "CHECK failed: nullptr \\(=\\(null\\)\\) != nullptr \\(=\\(null\\)\\)");
```

Comment:
I was expecting to see tests where you used quic logging macros passing in null strings and having them *not* crash, but instead spew out "(null)".

Reply:
oh I think I see what's going on...you are testing we spew 'nullptr' but then to get the log at all you have to ultimately crash? Isn't there the equivalent of a LOG(ERROR) you can capture?

Reply:
Good point! I've added more testing